.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+We, the rest of the NativeScript community, thank you for your
+feedback! 
+To help the rest of the community review your issue, please follow the instructions in the template.
+-->
+
+<!-- ISSUE TEMPLATE -->
+<!-- (Update "[ ]" to "[x]" to check a box) -->
+
+## Issue Checklist
+
+- [ ] You are submitting: <!-- a feature request; a bug report. For questions, please use the [NativeScript Forum](http://forum.nativescript.org). -->
+- [ ] You verified that this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the 
+[other issues in this repo](https://github.com/nativescript/nativescript-dev-webpack/issues?utf8=✓&q=is%3Aissue).
```

Comment:
The forums we use are located in https://discourse.nativescript.org/ . There is a redirect, but still you can change it to this link.